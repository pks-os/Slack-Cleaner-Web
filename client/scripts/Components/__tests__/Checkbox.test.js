import React from 'react';
import renderer from 'react-test-renderer';

import Checkbox from '../Checkbox';

const LABEL = 'LABEL TEXT';
// const mockCallback = jest.fn();

describe('<Checkbox />', () => {
  it('Renders Correctly', () => {
    const checkbox = renderer.create(<Checkbox />).toJSON();
    expect(checkbox).toMatchSnapshot();
  });

  it('Renders a label', () => {
    const checkbox = renderer.create(<Checkbox label={LABEL} />).toJSON();
    expect(checkbox).toMatchSnapshot();
  });
});
