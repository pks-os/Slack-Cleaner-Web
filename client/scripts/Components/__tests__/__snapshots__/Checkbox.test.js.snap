// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Checkbox /> Renders Correctly 1`] = `
<div
  className="Checkbox"
>
  <input
    checked={false}
    className="Checkbox__Input"
    id=""
    onChange={[Function]}
    type="checkbox"
    value=""
  />
  <label
    checked={false}
    className="Checkbox__Label"
    htmlFor=""
  >
    
  </label>
</div>
`;

exports[`<Checkbox /> Renders a label 1`] = `
<div
  className="Checkbox"
>
  <input
    checked={false}
    className="Checkbox__Input"
    id=""
    onChange={[Function]}
    type="checkbox"
    value=""
  />
  <label
    checked={false}
    className="Checkbox__Label"
    htmlFor=""
  >
    LABEL TEXT
  </label>
</div>
`;
